projects/1.16/assets/agricraft/agricraft/lang/zh_cn.json

```
@@ -0,0 +1,847 @@
+  "agricraft.plant.pumpkin.name": "南瓜",
+  "agricraft.plant.pumpkin.seed": "南瓜种子",
+  "agricraft.plant.pumpkin.desc": "如果吃的足够多我可以变成橙色吗？",
+
+  "agricraft.plant.red_mushroom.name": "红色蘑菇",
+  "agricraft.plant.red_mushroom.seed": "红色蘑菇孢子",
+  "agricraft.plant.red_mushroom.desc": "蘑菇，非常梦幻。",
+
+  "agricraft.plant.red_tulip.name": "红色郁金香",
+  "agricraft.plant.red_tulip.seed": "红色郁金香种子",
+  "agricraft.plant.red_tulip.desc": "红色的郁金香变种。这种植物的价值主要在于其独特的花朵。",
+
+  "agricraft.plant.sea_grass.name": "海草",
+  "agricraft.plant.sea_grass.seed": "海草种子",
+  "agricraft.plant.sea_grass.desc": "草，3000年前的草啊。",
+
+  "agricraft.plant.sea_pickle.name": "海泡菜",
+  "agricraft.plant.sea_pickle.seed": "海泡菜种子",
+  "agricraft.plant.sea_pickle.desc": "吃起来不像某人可能期望的。",
+
+  "agricraft.plant.sweet_berry.name": "甜浆果",
+  "agricraft.plant.sweet_berry.seed": "甜浆果种子",
+  "agricraft.plant.sweet_berry.desc": "你应该用它做个蛋糕。",
+
+  "agricraft.plant.sugarcane.name": "甘蔗",
+  "agricraft.plant.sugarcane.seed": "甘蔗种子",
+  "agricraft.plant.sugarcane.desc": "富含蔗糖，你日常早餐中非常健康的一部分。",
+
+  "agricraft.plant.warped_fungus.name": "诡异菌",
+  "agricraft.plant.warped_fungus.seed": "诡异菌孢子",
+  "agricraft.plant.warped_fungus.desc": "高效疣猪兽驱赶器.",
+
+  "agricraft.plant.wheat.name": "小麦",
+  "agricraft.plant.wheat.seed": "小麦种子",
+  "agricraft.plant.wheat.desc": "农业革命的基石。",
+
+  "agricraft.plant.white_tulip.name": "白色郁金香",
+  "agricraft.plant.white_tulip.seed": "白色郁金香种子",
+  "agricraft.plant.white_tulip.desc": "雪白的郁金香变种。这种植物的价值主要在于其独特的花朵。",
+
+  "agricraft.plant.wither_rose.name": "凋零玫瑰",
+  "agricraft.plant.wither_rose.seed": "凋零玫瑰种子",
+  "agricraft.plant.wither_rose.desc": "没人想要他们的玫瑰凋零，但这是另一层意义上的。",
+
+  "agricraft.plant.aurigold.name": "金光花",
+  "agricraft.plant.aurigold.seed": "金光花种子",
+  "agricraft.plant.aurigold.desc": "一种金灿灿的喜悦.",
+
+  "agricraft.plant.carbonation.name": "煤炭花",
+  "agricraft.plant.carbonation.seed": "煤炭花种子",
+  "agricraft.plant.carbonation.desc": "一种罕见的花，因为极其易燃导致自己濒临灭绝。",
+
+  "agricraft.plant.cuprosia.name": "红铜花",
+  "agricraft.plant.cuprosia.seed": "红铜花种子",
+  "agricraft.plant.cuprosia.desc": "被称为工程师的喜悦，这种植物产出效果惊人的物质。",
+
+  "agricraft.plant.diamahlia.name": "金刚大丽",
+  "agricraft.plant.diamahlia.seed": "金刚大丽种子",
+  "agricraft.plant.diamahlia.desc": "无论如何都是罕见的。",
+
+  "agricraft.plant.emeryllis.name": "朱顶绿",
+  "agricraft.plant.emeryllis.seed": "朱顶绿种子",
+  "agricraft.plant.emeryllis.desc": "真正的绿宝石",
+
+  "agricraft.plant.ferranium.name": "铁叶草",
+  "agricraft.plant.ferranium.seed": "铁叶草种子",
+  "agricraft.plant.ferranium.desc": "一种顽强的植物。",
+
+  "agricraft.plant.jaslumine.name": "茉莉",
+  "agricraft.plant.jaslumine.seed": "茉莉种子",
+  "agricraft.plant.jaslumine.desc": "茉莉赠予人，茉莉随我身。灼灼如银华，草木意深深。",
+
+  "agricraft.plant.lapender.name": "蓝熏衣",
+  "agricraft.plant.lapender.seed": "蓝熏衣种子",
+  "agricraft.plant.lapender.desc": "一种华贵的植物。",
+
+  "agricraft.plant.niccissus.name": "镍粉藤",
+  "agricraft.plant.niccissus.seed": "镍粉藤种子",
+  "agricraft.plant.niccissus.desc": "虽然关于哪种植物最好还有很多争论，我们都认同镍粉藤是最自私的植物。",
```

```suggestion
  "agricraft.plant.niccissus.desc": "虽然关于哪种植物最适合“自私”的称号还有很多争论，但直到现在，我们都认同镍水仙是最自私的。",
```
  "agricraft.plant.niccissus.desc": "While there is much debate over which is the best plant, we can all agree that the Niccissus plant is by far the most self-absorbed.",